src/components/Cart.js

```diff
@@ -28,7 +26,7 @@ const Cart = () => {
           <h3>Total</h3>
         </div>
         <div>
-          <span className="total-number">£0.00</span>
+          <span className="total-number">{total()}</span>
```

Comment:
does the total display £0.00 before items are added to cart?

Reply:
I don't think so Kiran.... maybe he needs to implement toFixed in there.